docs/source/en/model_doc/albert.md

```diff
@@ -14,100 +14,144 @@ rendered properly in your Markdown viewer.
 
-#### Inference with 50 batches
 
-|batch_size|seq_len|Per token latency eager (ms)|Per token latency SDPA (ms)|Speedup (%) |Mem eager (MB)|Mem BT (MB)|Mem saved (%)|
-|----------|-------|----------------------------|---------------------------|------------|--------------|-----------|-------------|
-|4         |128    |0.083                       |0.071                      |16.967      |48.319        |48.45      |-0.268       |
-|4         |256    |0.148                       |0.127                      |16.37       |63.4          |63.922     |-0.817       |
-|4         |512    |0.31                        |0.247                      |25.473      |110.092       |94.343     |16.693       |
-|8         |128    |0.137                       |0.124                      |11.102      |63.4          |63.66      |-0.409       |
-|8         |256    |0.271                       |0.231                      |17.271      |91.202        |92.246     |-1.132       |
-|8         |512    |0.602                       |0.48                       |25.47       |186.159       |152.564    |22.021       |
-|16        |128    |0.252                       |0.224                      |12.506      |91.202        |91.722     |-0.567       |
-|16        |256    |0.526                       |0.448                      |17.604      |148.378       |150.467    |-1.388       |
-|16        |512    |1.203                       |0.96                       |25.365      |338.293       |271.102    |24.784       |
+```py
+from transformers import AutoModelForMaskedLM, AutoTokenizer
+import torch
+
+# Load model ---loaded v1 version as it was answering good
+model = AutoModelForMaskedLM.from_pretrained("albert/albert-base-v1")
+tokenizer = AutoTokenizer.from_pretrained("albert/albert-base-v1")
+
+# Quantize the model (PyTorch native)
+quantized_model = torch.quantization.quantize_dynamic(
+    model,
+    {torch.nn.Linear},  # Quantize only linear layers
+    dtype=torch.qint8
+)
+
+# Verify
+print(f"Size before: {model.get_memory_footprint()/1e6:.1f}MB")
+print(f"Size after: {quantized_model.get_memory_footprint()/1e6:.1f}MB")
+
+# Usage example
+inputs = tokenizer("Albert Einstein was born in [MASK].", return_tensors="pt")
+with torch.no_grad():
+    outputs = quantized_model(**inputs)
+    print(tokenizer.decode(outputs.logits[0].argmax(-1)))
+```
+
+> ALBERT is not compatible with `AttentionMaskVisualizer` as it uses masked self-attention rather than causal attention. So don't have `_update_causal_mask` method.
 
-This model was contributed by [lysandre](https://huggingface.co/lysandre). This model jax version was contributed by
-[kamalkraj](https://huggingface.co/kamalkraj). The original code can be found [here](https://github.com/google-research/ALBERT).
+## Notes
 
+- All tokens attend to all others (like BERT).
```

Comment:
```suggestion
- Inputs should be padded on the right because BERT uses absolute position embeddings.
- The embedding size `E` is different from the hidden size `H` because the embeddings are context independent (one embedding vector represents one token) and the hidden states are context dependent (one hidden state represents a sequence of tokens). The embedding matrix is also larger because `V x E` where `V` is the vocabulary size. As a result, it's more logical if `H >> E`. If `E < H`, the model has less parameters.
```